templates/sectionsummary.html

```diff
@@ -31,7 +31,7 @@ <h2>{{summary.title}}</h2>
           {% if summary.list.list_items %}
             {% set list = summary.list %}
             {% include 'partials/summary/list-summary.html' %}
-            {% else %}
+            {% elif summary.empty_list_text %}
```

Comment:
It's minor, but the indentation is off here, and as we're making changes, we should fix it. The `elif` should align with the `if` (and it looks like everything below it down to the `endfor` needs to be shifted left).

Reply:
Thanks, fixed now!